pkg/controllers/machinesync/machine_sync_controller_test.go

```diff
@@ -738,6 +754,276 @@ var _ = Describe("With a running MachineSync Reconciler", func() {
 			})
 		})
 	})
+
+	Context("validating admission policy tests", func() {
+		var (
+			policyBinding *admissionregistrationv1.ValidatingAdmissionPolicyBinding
+			machineVap    *admissionregistrationv1.ValidatingAdmissionPolicy
+		)
+
+		BeforeEach(func() {
+			By("Loading the transport config maps")
+			transportConfigMaps := admissiontestutils.LoadTransportConfigMaps()
+
+			By("Applying the objects found in clusterAPICustomAdmissionPolicies")
+			for _, obj := range transportConfigMaps[admissiontestutils.ClusterAPICustomAdmissionPolicies] {
+				// Deep‑copy so the loop variable isn’t mutated by the client
+				newObj, ok := obj.DeepCopyObject().(client.Object)
+				Expect(ok).To(BeTrue())
+
+				err := k8sClient.Create(ctx, newObj)
+				if err != nil && !apierrors.IsAlreadyExists(err) {
+					Expect(err).NotTo(HaveOccurred())
+				}
```

Comment:
Should we use Eventually? Not sure how feasible it is with the already exists error being there

Reply:
Have updated, I think it works :D